packages/web-components/src/search/product-search-overlay.ts

```diff
@@ -161,6 +161,11 @@ export class ProductSearchOverlay extends LitElement {
                 .setExplodedVariants(searchOptions?.explodedVariants ?? defaultExplodedVariants)
                 .setTerm(searchTerm)
                 .pagination(p => p.setPageSize(this.numberOfProducts))
+                .relevanceModifiers(builder => {
```

Comment:
Should we have a common basebuilder like we have for recommendations, so we dont need to "remember" to add stuff both places?